src/main/scala/cromwell/binding/WdlBinding.scala

```diff
@@ -404,7 +409,10 @@ case class WdlBinding(ast: Ast, source: WdlSource, importResolver: ImportResolve
         val task = findTask(taskName) getOrElse {
           throw new SyntaxError(wdlSyntaxErrorFormatter.callReferencesBadTaskName(callAst, taskName))
         }
-        val callName = if(callAst.getAttribute("alias") != null) sourceString(callAst.getAttribute("alias")) else task.name
+        val callName =
+          if(callAst.getAttribute("alias") != null) sourceString(callAst.getAttribute("alias"))
+          else sourceString(callAst.getAttribute("task"))
```

Comment:
Perhaps something like:

    val source = Option(callAst.getAttribute("alias") getOrElse callAst.getAttribute("task")
    sourceString(source)

(or all in one line if that's your fancy)

We should avoid direct references to `null` as much as possible.